code/src/components/AddItem.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { useDispatch } from "react-redux"
+import styled from 'styled-components'
+import { todo } from '../reducers/todo'
+
+const InputText = styled.input`
+  width: 100%;
+  padding: 8px 10px;
+  margin: 8px 0;
+  display: inline-block;
+  border: none;
+  border-bottom: 1px solid black;
+  box-sizing: border-box;
+  background-color: DodgerBlue;
+
+  ::placeholder {
+    color: black;
+  }
+`
+
+const Select = styled.select`
+  width: 100%;
+  padding: 8px 10px;
+  margin: 8px 0;
+  display: inline-block;
+  border: none;
+  border-bottom: 1px solid black;
+  box-sizing: border-box;
+  background-color: DodgerBlue;
+`
+
+const InputBtn = styled.input`
+  width: 100%;
+  background-color: DeepPink;
+  color: white;
+  font-weight: bold;
+  padding: 8px 10px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+
+  &:hover {
+    background-color: #e23290;
+  }
+`
+
+export const AddItem = () => {
+  const dispatch = useDispatch()
+  const [inputValue, setInputValue] = useState('')
+  const [category, setCategory] = useState('other')
+
+  const handleSubmit = e => {
+    e.preventDefault()
+
+    const date = new Date()
+
+    dispatch(
+      todo.actions.addTodo({
+        itemInfo: {
+          id: Date.now(),
+          description: inputValue,
+          category: category,
+          startDate: date.toLocaleDateString( 'se-SE', { dateStyle: 'short' }),
+          dueDate: 2021
+        }
+      })
+    )
+
+    setInputValue('')
+  }
+
```

Comment:
Nice!
Two things here, you could set all the fields that are not selected by the user (such as id, startDate) in the reducer instead of here. Just passing the user inputs from this component. But this is totally fine!
The other thing is that you right now can add an empty todo. This you could easily check either by adding this line to the top of the handleSubmit function:
`if (!inputValue) return` Which will return from the function (stop it) if there is no input value. 
You could also make the input value a required field in the form or disable the submit button until something is put into the input.